app/src/main/java/com/example/beetrackerrough/BeeSightingUserInput.kt

```diff
@@ -1,13 +1,35 @@
 package com.example.beetrackerrough
 
+import android.annotation.SuppressLint
+import android.app.Activity.RESULT_OK
+import android.content.Intent
+import android.net.Uri
 import android.os.Bundle
+import android.os.Environment
+import android.provider.MediaStore
+import android.telephony.MbmsDownloadSession.RESULT_CANCELLED
```

Comment:
For some reason `android.app.Activity,RESULT_CANCELLED` doesn't exist, or according to Android Studio at least.

Reply:
I think the request code is RESULT_CANCELED (with 1 L).  I updated it and will try running it.